src/schema/ecommerce/types/offer.ts

```diff
@@ -67,6 +72,11 @@ export const OfferType = new GraphQLObjectType({
       type: OrderInterface,
       description: "The order on which the offer was made",
     },
+    buyerTotalCents: {
+      type: GraphQLInt,
+      description: "Offer amount in cents",
```

Comment:
Also might be good to specify in the description that buyerTotalCents includes tax and shipping.